src/Umbraco.Core/Services/MemberTypeService.cs

```diff
@@ -16,8 +14,8 @@ public class MemberTypeService : ContentTypeServiceBase<IMemberTypeRepository, I
         private readonly IMemberTypeRepository _memberTypeRepository;
 
         public MemberTypeService(ICoreScopeProvider provider, ILoggerFactory loggerFactory, IEventMessagesFactory eventMessagesFactory, IMemberService memberService,
-            IMemberTypeRepository memberTypeRepository, IAuditRepository auditRepository, IEntityRepository entityRepository, IEventAggregator eventAggregator)
-            : base(provider, loggerFactory, eventMessagesFactory, memberTypeRepository, auditRepository, null, entityRepository, eventAggregator)
+            IMemberTypeRepository memberTypeRepository, IAuditRepository auditRepository, IMemberTypeContainerRepository entityContainerRepository, IEntityRepository entityRepository, IEventAggregator eventAggregator)
+            : base(provider, loggerFactory, eventMessagesFactory, memberTypeRepository, auditRepository, entityContainerRepository, entityRepository, eventAggregator)
```

Comment:
Breaking change

Reply:
True... are you concerned though with being breaking from rc1, or from v9?  Either way though, aren't we allowed a change here (i.e. in a constructor) given we've called out that there may be changes there in the blog/release notes?

Reply:
If possible I think we should try to add an obsolete ctor pointing to the new

Reply:
OK, I've added this.